src/app/components/onc-organization/onc-organization-view.jsx

```diff
@@ -74,14 +66,38 @@ function ChplOncOrganizationView(props) {
              />
            </Typography>
          )}
-        { organization.address
+        </Box>
+        <Box className={classes.subContentBox}>
+          <Typography variant="body1" gutterBottom>
+            <strong>ONC-ACB code</strong>
+          </Typography>
+          <Typography>{ organization.acbCode}</Typography>
+        </Box>
+        <Box className={classes.subContentBox}>
+          <Typography variant="body1" gutterBottom>
+            <strong>Retired</strong>
+          </Typography>
+          { organization.retired ? 'Yes' : 'No' }
+        </Box>
+        <Box className={classes.subContentBox}>
+          {' '}
+          { organization.retired
+            && (
+              <>
+                <Typography variant="body1" gutterBottom><strong>Retirement Date</strong></Typography>
+                { getDisplayDateFormat(organization.retirementDay) }
+              </>
+            )}
+        </Box>
+        <Box className={classes.subContentBox}>
+          { organization.address
          && (
            <Typography variant="body1" gutterBottom>
              <strong>Address</strong>
              <br />
              <span className="sr-only">Line 1: </span>
-             {organization.address.line1}
-             {organization.address.line2
+             { organization.address.line1}
```

Comment:
I appreciate the spacing here, but it's both not necessary (as this is just a single element being displayed), and inconsistent, as the ending `}` doesn't have a space. I think adding this space isn't needed, and that's true with the other fields below